src/Saturn.Extensions.gRPC/Saturn.gRPC.fs

```diff
@@ -21,6 +21,6 @@ type Saturn.Application.ApplicationBuilder with
             app.UseEndpoints (fun endpoints -> endpoints.MapGrpcService<'a>() |> ignore)
 
         { state with
-            AppConfigs = configureApp::configureGrpcEndpoint::state.AppConfigs
+            AppConfigs = configureApp::configureGrpcEndpoint::state.AppConfigs.[1.. state.AppConfigs.Length]
```

Comment:
This sounds risky - as far as I understand it'd potentially break if the `use_grpc` calls were not in a row.

For example, if I had in my `application` following code it would remove the wrong entry in `state.AppConfigs`:
```
use_grpc MyHelloWithDI
use_static
use_grpc MyCalculatorWithDI
```

TBF, I'm not yet sure what's the good solution here.

Reply:
I was about to open an issue when I found this PR. I know this is not a solution but a workaround that worked for me was:

```fsharp
type MyReflectionServiceImpl(descriptors: Google.Protobuf.Reflection.ServiceDescriptor seq) =
  inherit ReflectionServiceImpl(descriptors)

let app =
  application {
    use_grpc MyReflectionServiceImpl
   // other config removed for clarity
  }

```